tests/test_views.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from flask import render_template_string, g
+
+from udata.core.dataset.factories import DatasetFactory
+
+
+def render_hook(dataset):
+    return render_template_string(
+        '{{ hook("dataset.display.transport-banner") }}',
+        dataset=dataset
+    )
+
+
+@pytest.fixture
+def datasets():
+    return DatasetFactory.create_batch(3)
+
+
+@pytest.mark.frontend
+@pytest.mark.usefixtures('clean_db')
+@pytest.mark.usefixtures('app')
+@pytest.mark.options(THEME='gouvfr')
```

Comment:
It is not need to add this theme configuration I think?

Reply:
the `theme.render` seem to need it